crates/viewer/re_viewer/src/ui/open_url_modal.rs

```diff
@@ -0,0 +1,104 @@
+use std::str::FromStr as _;
+
+use re_ui::modal::{ModalHandler, ModalWrapper};
+use re_ui::{UICommand, UiExt as _};
+
+use crate::open_url::ViewerOpenUrl;
+
+#[derive(Default)]
+pub struct OpenUrlModal {
+    modal: ModalHandler,
+    url: String,
+    just_opened: bool,
+}
+
+impl OpenUrlModal {
+    pub fn open(&mut self) {
+        self.modal.open();
+        self.just_opened = true;
+    }
+
+    pub fn ui(&mut self, ui: &egui::Ui) {
+        self.modal.ui(
+            ui.ctx(),
+            || ModalWrapper::new("Open from URL").max_width(400.0),
+            |ui| {
+                ui.horizontal(|ui| {
+                    ui.label(
+                        egui::RichText::new("Paste a URL below.")
+                            .color(ui.visuals().strong_text_color()),
+                    );
```

Comment:
I think?

```suggestion
                    ui.strong("Paste a URL below.")
```

Reply:
yeah agreed, the color only is weird.
I wanted to get closer to Katya's design where this is "quite strong" but not bold